### DISCUSSION TITLE
Header accessibility issues

### DISCUSSION THREAD

**Discussion by brunnerh** [revolist/revogrid#450] [General]
**Describe the issue**
- The header elements cannot be focused via the keyboard through Tab. Sortable headers should be focusable and behave as if the user clicked on them upon pressing Enter.
- The filter buttons in the header are focusable but their visibility is not tied to focus. They are only visible upon mouse hover.

**To Reproduce**
[[Minimal example]](https://codesandbox.io/s/revo-grid-accessibility-43y84)
I added a style to add a red border on the header cells when the focus is on a filter button within the cell.

- Tab around, the header itself will never receive focus.
- Tab until a filter button receives focus. It will only become visible on mouse hover.

**Comment by revolist** [comment]
Can be done as part of WCAG plugin for Pro version.

**Reply by Jordan-Hall** [reply]
I've got the pro but i cant seem to import WCAG plugin as its not exported?

**Reply by revolist** [reply]
Hi, it's an improvement to make, we have added this to our pro version roadmap already
